Apps/W1/File - Azure Blob Service Connector/app/app.json

```
@@ -0,0 +1,30 @@
+{
+    "id":  "c9ce86fe-cb70-4b79-be03-d21856b1a4ca",
+    "name":  "File - Azure Blob Service Connector",
```

Let's align the naming:

```suggestion
    "name":  "External File Storage - Azure Blob Service Connector",
```